### PR TITLE
Move docker-commons taglib to /lib/docker/commons

### PR DESCRIPTION
**Pull request by oleg-nenashev**
As @ndeloof suggested

@reviewbybees

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerTool.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
 /**
  * An installation of Docker.
  * If nonempty, expected to have an executable file {@code bin/docker}.
- * Use {@code <d:selectDockerTool field="toolName" xmlns:d="/org/jenkinsci/plugins/docker/commons/lib"/>} to customize.
+ * Use {@code <d:selectDockerTool field="toolName" xmlns:d="/lib/docker/commons/"/>} to customize.
```
Comment (ndeloof) [inline src/main/java/org/jenkinsci/plugins/docker/commons/tools/DockerTool.java]: no need for a trailing `/` in namespace afaik

**Comment by ndeloof** [comment]
:+1:

**Comment by tfennelly** [comment]
:+1:

**Comment by jglick** [comment]
While it is too late now, FTR: I intentionally used the same package namespace as the plugin itself. IMO only core should be able to use the `/lib` namespace. But the only precedent that I know of is in Credentials.

**Comment by oleg-nenashev** [comment]
@jglick 
I have no strong opinion on this change. We could deprecate the existing taglib and create it's copy at the right location.

**Pull request merged**